build_system/ninja_utils.mli

```diff
@@ -0,0 +1,108 @@
+(* This file is part of the Catala build system, a specification language for tax and social
+   benefits computation rules. Copyright (C) 2020 Inria, contributor: Emile Rolley
+   <emile.rolley@tuta.io>
+
+   Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+   in compliance with the License. You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software distributed under the License
+   is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+   or implied. See the License for the specific language governing permissions and limitations under
+   the License. *)
+
+(** {1 Ninja expressions} *)
```

Comment:
A little documentation header here reminding what is ninja, where is the manual and the basic concepts of ninja would be useful.